tests/test_views_objects_permissions.py

```diff
@@ -103,6 +103,16 @@ def test_cannot_write_if_not_allowed(self):
                           headers=self.alice_headers,
                           status=403)
 
+    def test_permission_backend_prevent_sql_injections(self):
+        self.app.get("/buckets/beer'", headers=self.headers, status=403)
+        self.app.get("/buckets/beer'/collections/barley", headers=self.headers, status=403)
+        self.app.get("/buckets/beer'/groups/barley", headers=self.headers, status=403)
+
+        self.app.get("/buckets/beer/collections/barley'", headers=self.headers, status=400)
+        self.app.get("/buckets/beer/collections/barley'/records", headers=self.headers, status=404)
```

Comment:
Oh no its not a bug because we have the permissions on the beer bucket so it makes sense.